portfolio/src/main/java/com/google/sps/servlets/ContactInformationServlet.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+//import org.jsoup.Jsoup;
+//import org.jsoup.safety.Whitelist;
+
+@WebServlet("/contact-information")
+public class ContactInformationServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get first name entered in the form.
+    String firstName = getParameter(request, "fname", "");
+    
+    // Get last name entered in the form.
+    String lastName = getParameter(request, "lname", "");
+    
+    // Get the name entered in the form.
+    String email = getParameter(request, "email", "");
+
+    // Reason of contacting
+    boolean academicContact = Boolean.parseBoolean(getParameter(request, "academic-contact", "false"));
+    boolean professionalContact = Boolean.parseBoolean(getParameter(request, "professional-contact", "false"));
+    boolean otherContact = Boolean.parseBoolean(getParameter(request, "other-contact", "false"));
+    String otherContactReason = getParameter(request, "other-contact-reason", "");
+    String contactReason = "";
+
+    System.out.println("New contact: " + firstName + " " + lastName + ", " + email);
+    if (academicContact) {
```

Comment:
Use a switch with fallthrough and print once at the end

Reply:
Not sure how I would be able to do that since I have ore than one variable for the "reason of contacting". Also, the PR indicates there are conflicts with the main branch, and it's just a few lines of code, but I don't know how to fix it. I tried but it keeps saying there is conflict.